homeassistant/helpers/typing.py

```diff
@@ -4,6 +4,22 @@
 
 import homeassistant.core
 
+__all__ = (
```

Comment:
I don't think we should add this. We should only use explicit imports. Wildcard imports are terrible for code readability.

Reply:
I wasn't completely sure about it. I imagined that it might make sense to do
```py
from homeassistant.helpers.typing import *
```
to import some frequently used types.

--
I'll remove that commit.